elytron/src/main/java/org/wildfly/extension/elytron/TrivialResourceDefinition.java

```diff
@@ -173,6 +182,11 @@ Builder setRuntimeCapabilities(RuntimeCapability<?> ... runtimeCapabilities) {
             return this;
         }
 
+        Builder setStability(Stability stability) {
+            this.stability = stability;
+            return this;
+        }
+
```

Comment:
This seems redundant.  The super implementation already defines a stability field.